### PR TITLE
Change title

### PR DESCRIPTION
**Pull request by demomemoemo**
me create pull request ok to change title

### CODE REVIEW

**Comment by demomemoemo** [comment]
hello

**Comment by demomemoemo** [comment]
my name is demomemoemo

**Comment by aquafreq** [comment]
в8

File: HTML & CSS Exam - 28 March 2020/02. Beats/index.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
             <div class="inner">
               <p><span class="uppercase">solo wireless</span></p>
               <p>
-                With up to 40 hours of battery life, Beats Solo Wireless is your
+                With up to 40 hours of BIMBO!!, Beats Solo Wireless is your
```
Comment (aquafreq) [inline HTML & CSS Exam - 28 March 2020/02. Beats/index.html:33]: why u troll

File: HTML & CSS Exam - 28 March 2020/02. Beats/index.html [hunk 2]
```diff
@@ -29,7 +29,7 @@
             <div class="inner">
               <p><span class="uppercase">solo wireless</span></p>
               <p>
-                With up to 40 hours of battery life, Beats Solo Wireless is your
```
Comment (aquafreq) [inline HTML & CSS Exam - 28 March 2020/02. Beats/index.html:32]: why remove ?

**Review by aquafreq — CHANGES_REQUESTED** [review]
needs more work, ok?

File: HTML & CSS Exam - 28 March 2020/02. Beats/index.html [hunk 3]
```diff
@@ -30,6 +30,7 @@
               <p><span class="uppercase">solo wireless</span></p>
               <p>
                 changegin title :))With up to 40 hours of battery life, Beats Solo Wireless is your
+                With up to 40 hours of BIMBO!!, Beats Solo Wireless is your
                 perfect everyday headphone. Get the most out of your music with
                 an award-winning, emotionally charged Beats listening
```
Comment (aquafreq) [inline HTML & CSS Exam - 28 March 2020/02. Beats/index.html:35]: wtf?

**Pull request merged**